xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -6348,7 +6348,8 @@ fuse_priv_dump(xlator_t *this)
     if (!this)
         return -1;
 
-    private = this->private;
+   private
+    = this->private;
```

Comment:
This doesn't make sense. Probably clang-format does this assuming this is C++ code. We should change clang-format configuration to avoid this kind of formatting.

Reply:
Yes, actually these errors don't show up in my local setup.

I'm just annoyed that clang-format is always nagging us & disrupts the review flow with a signifcant amount of noise whenever fuse-bridge.c is touched.

It's also fine with me if the report is hushed on the CI side. Just be it hushed ;)

Reply:
If we completely remove the output from the comments, then nobody will care about it. But I agree that it's showing a lot of unwanted noise. I'll try to take a look on how to improve it.

Reply:
OK so do you think it's fine if I just go ahead with the glusterfs.h empty line removal fix (adjusting the subject accordingly)? This is frowned upon my local clang-format as well.